src/core/mormot.core.base.pas

```diff
@@ -110,6 +110,9 @@ interface
   /// use rather CP_LATIN1 with mORMot 2
   CODEPAGE_LATIN1 = CP_LATIN1;
 
+  /// Maximum bytes for single UTF-8 char.

```

Comment:
It is not the right place to put this.
And it is a very difficult guess to make, because we support up to 6 bytes per Unicode point, which theoretically could be 32-bit.
Note that there is no such this as an "UTF-8 char".
So I would not define such a constant.

Reply:
Ok. I will change this.